Scripts/script-CrowdStrikeStreamingPreProcessing.yml

```diff
@@ -0,0 +1,39 @@
+commonfields:
+  id: CrowdStrikeStreamingPreProcessing
+  version: -1
+name: CrowdStrikeStreamingPreProcessing
+script: |-
+  import sys
+
+  res=False
```

Comment:
Default should be true and turn to false only if you find dup. Also, add back spaces :-)